src/styles/_category.scss

```
@@ -1,54 +1,313 @@
+@use './mixins';
+
 /***** Category pages *****/
-.category {
-  &-container {
-    display: flex;
-    justify-content: flex-end;
+@media only screen and (min-width: 320px) {
+  .hero__lockup {
+    grid-template-areas: 'icon' 'title' 'subtitle' 'search';
```

This isn't a bug or anything, just a coding tip that I learned at a workshop by Rachel Andrew. She recommended arranging template areas in your code in a way that represents the layout of the grid:

```suggestion
    grid-template-areas:
      'icon'     
      'title'
      'subtitle' 
      'search';
```

(See other comment for subsequent arrangement)

I prefer this way of writing grid template areas because it makes it easier for the me and subsequent contributors to reason about and understand how the code relates to the design.

See also [Understanding CSS Grid: Grid Template Areas by Rachel Andrews](https://www.smashingmagazine.com/understanding-css-grid-template-areas/)